NewForce-Cohort-7/tabloidfullstack-numpty-buggers
# hotfix for DeleteCategory

### Description

This is a hotfix to PR #8 (Delete a Category)

### Update

Changed the code to ensure that the user is redirected to the correct page after clicking "Yes" on the "Delete" page.

### Type of Change


### Testing Instructions

1. Click `Category Management` in the navbar.
2. Click the blue `Create Category` button at the bottom of the list.
3. Enter a name for your new category (eg. Gaming) then click `SUBMIT`.
4. Refresh the list of categories to see if your item was added.
5. Click the red `Delete` button, you should be redirected to `/category/delete/{id}`
6. When prompted, click the `Yes` button, you should be redirected to the list of categories.
